spec.md

```diff
@@ -1177,6 +1177,18 @@ which MUST be implemented as described if supported:
     ipc: "service:[service name]"
 ```
 
+### uts
+
+`uts` configures the UTS namespace mode set for the service container. Available values are:
+
+- `''` which results in the container having it's own UTS namespace.
```

Comment:
to align with other attributes, empty value should not be documented (i.e: this is up to container runtime to assign an UTS namespace, depending on it's implementation details and configuration).
see https://github.com/compose-spec/compose-spec/blob/master/spec.md#cgroup for illustration